README.md

```diff
@@ -49,7 +49,7 @@ You should now have the following directories at the same level somewhere:
 
 ```
 example.com/    - Primary folder for the project
-├── ansible/    - Your version of this repo (renamed to just `ansible`)
+├── trellis/    - Your version of this repo (renamed to just `trellis`)
```

Comment:
While we are at it: 

"Your version of this repo (renamed to just `trellis`)" :arrow_right: "Your clone of this repo"